lua/telescope/actions.lua

```diff
@@ -117,21 +124,35 @@ local function goto_file_selection(prompt_bufnr, command)
   end
 end
 
-function actions.goto_file_selection_edit(prompt_bufnr)
-  goto_file_selection(prompt_bufnr, "edit")
-end
-
-function actions.goto_file_selection_split(prompt_bufnr)
-  goto_file_selection(prompt_bufnr, "new")
-end
-
-function actions.goto_file_selection_vsplit(prompt_bufnr)
-  goto_file_selection(prompt_bufnr, "vnew")
-end
-
-function actions.goto_file_selection_tabedit(prompt_bufnr)
-  goto_file_selection(prompt_bufnr, "tabedit")
-end
+actions.center = setmetatable({
```

Comment:
instead of doing this, I think at the end of actions, right before we return, we should do something like the following:

```lua
for k, v in pairs(actions) do
  actions[k] = setmetatable({__call = function(_, ...) return k(...) }, action_mt)
end
```

Note, I wrote this code in the github review thing, so it probably doesn't work :laughing:

Reply:
and then after that we can do `actions._action_mt = action_mt` or something like that.

Does that make sense?

Reply:
Yeah makes sense and makes things quiet easy